src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--mon calendar--month-length--30">
+      <div class="calendar__month">
```

Comment:
i think it will be better to use semantic tags

Reply:
Thanks! I forgot about that.